src/reorderable-repeat.js

```diff
@@ -217,16 +217,17 @@ export class ReorderableRepeat extends AbstractRepeater {
   unbind() {
     this.groupMap.remove(this);
 
-    this.scope = null;
-    this.items = null;
-    this.matcherBinding = null;
-    this.viewSlot.removeAll(true);
     if (this.arrayObserver) {
       this.arrayObserver.dispose();
       this.arrayObserver = null;
     }
     this._subsribers.forEach(s => s.dispose());
     this._subsribers = [];
+
+    this.removeAllViews(true, !this.viewsRequireLifecycle);
```

Comment:
@bigopon, following the code of original repeat, here I guess we need skipAnimation in unbind?
Like this:
```js
this.removeAllViews(true, true);
```
Another question is why "unbind" need to deal with removing views? Should not view be handled in "detached" lifecycle?

Reply:
@3cp
> why "unbind" needs to deal with removing views? ... handled in "detached" lifecycle?
Detached and unbind usually goes together, so it's quite the same, that's my feeling. I haven't encountered a case where detached would be invoked while `unbind` would not.